care/facility/api/serializers/testing_lab.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348729)